File: Python/distance_calc.py
from math import sqrt

def distance2d(_from, to):
	diffX = abs(_from[0] - to[0])
	diffY = abs(_from[1] - to[1])
	delta = sqrt((diffX ** 2) + (diffY ** 2))
	return delta
	
def distance3d(_from, to):
	from2d = (_from[0], _from[2])
	to2d = (to[0], to[2])
	diffX = distance2d(from2d, to2d)
	diffY = abs(_from[1] - to[1])
	delta = sqrt((diffX ** 2) + (diffY ** 2))
	return delta
	
if __name__ == "__main__":
	a = (1, 2, 3)
	b = (4, 5, 6)
	distance = distance3d(a, b)
	print(distance)
	if distance > 5.0:
		exit(0)
	else:
		exit(1)

File: Kotlin/DistanceCalc.kt
/*
** Fun Fact!
** There's a Polish ketchup company called "Kotlin"
** QeaML, 10.02.2020
*/

fun distance2d(from: Array<Double>, to: Array<Double>): Double{
    val diffX = Math.abs(from[0] - to[0])
    val diffY = Math.abs(from[1] - to[1])
    val delta = Math.sqrt(Math.pow(diffX, 2.0) + Math.pow(diffY, 2.0))
    return delta
}

fun distance3d(from: Array<Double>, to: Array<Double>): Double{
    val from2d = Array(2){from[0]; from[2]}
    val to2d = Array(2){to[0]; to[2]}
    val diffX = distance2d(from2d, to2d)
    val diffY = Math.abs(from[1] - to[1])
    val delta = Math.sqrt(Math.pow(diffX, 2.0) + Math.pow(diffY, 2.0))
    return delta
}

fun main() {
    val a = Array(3){1.0; 2.0; 3.0}
    val b = Array(3){3.0; 4.0; 6.0}
    println(distance3d(a, b))
}